<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>708</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="gbControl">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="layControl">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gbPort">
         <property name="title">
          <string>Port</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="cbPort"/>
          </item>
          <item>
           <widget class="QComboBox" name="cbBaudrate"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Classic</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QVBoxLayout" name="laySettings_1">
                <item>
                 <layout class="QHBoxLayout" name="laySettingsCb_1">
                  <item>
                   <widget class="QCheckBox" name="cbEnabled_1">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/icons/red-circle.png</normaloff>:/icons/red-circle.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>32</width>
                      <height>32</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cbSign_1">
                    <property name="currentText">
                     <string>signed</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>signed</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>unsigned</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLineEdit" name="leName_1"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="laySettings_2">
                <item>
                 <layout class="QHBoxLayout" name="laySettingsCb_2">
                  <item>
                   <widget class="QCheckBox" name="cbEnabled_2">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/icons/blue-circle.png</normaloff>:/icons/blue-circle.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>32</width>
                      <height>32</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cbSign_2">
                    <property name="currentText">
                     <string>signed</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>signed</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>unsigned</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLineEdit" name="leName_2"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="laySettings_3">
                <item>
                 <layout class="QHBoxLayout" name="laySettingsCb_3">
                  <item>
                   <widget class="QCheckBox" name="cbEnabled_3">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/icons/green-circle.png</normaloff>:/icons/green-circle.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>32</width>
                      <height>32</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cbSign_3">
                    <property name="currentText">
                     <string>signed</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>signed</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>unsigned</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLineEdit" name="leName_3"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="laySettings_4">
                <item>
                 <layout class="QHBoxLayout" name="laySettingsCb_4">
                  <item>
                   <widget class="QCheckBox" name="cbEnabled_4">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/icons/yellow-circle.png</normaloff>:/icons/yellow-circle.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>32</width>
                      <height>32</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cbSign_4">
                    <property name="currentText">
                     <string>signed</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>signed</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>unsigned</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLineEdit" name="leName_4"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Extended</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbSend">
         <property name="title">
          <string>Send</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="leSendVar_1"/>
          </item>
          <item>
           <widget class="QLineEdit" name="leSendVar_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnSend">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuProtocol">
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Protocol</string>
     </property>
     <addaction name="actionClassic"/>
     <addaction name="actionTdfp"/>
    </widget>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionWhite"/>
     <addaction name="actionBlack"/>
    </widget>
    <addaction name="menuProtocol"/>
    <addaction name="menuTheme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClassic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Classic</string>
   </property>
  </action>
  <action name="actionTdfp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tdfp</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
